src/main/java/org/hyperledger/fabric/sdk/BlockInfo.java

```diff
@@ -31,6 +31,7 @@
 import org.hyperledger.fabric.protos.peer.TransactionPackage;
 import org.hyperledger.fabric.sdk.exception.InvalidProtocolBufferRuntimeException;
 import org.hyperledger.fabric.sdk.transaction.ProtoUtils;
+import sun.jvm.hotspot.utilities.AssertionFailure;
```

Comment:
This is failing the build and will prevent it running on non-Oracle/Sun JVMs, so this import can be used

Reply:
Removed. Error on my end, ignored the auto-IDE imports during commit.